doc/publish_intents.md

```diff
@@ -17,7 +17,6 @@ propagated out.
 
 A publish intent has the following fields:
 
-* `base_path` - The path of the item that will be published.
```

Comment:
I'd leave this here, but indicate that it's taken from the URL when writing, and included in the JSON when reading.